core/shortcut_items.js

```diff
@@ -47,8 +47,8 @@ const registerEscape = function() {
     preconditionFn: function(workspace) {
       return !workspace.options.readOnly;
     },
-    callback: function() {
-      Blockly.hideChaff();
+    callback: function(workspace) {
```

Comment:
You have verified that whatever calls this callback actually supplies a Workspace as the first argument?

Reply:
Yes, line 239 in shortcut_registry.js does this; the callbacks in this file just omit it (and the other args) if they don't care about them.